VOUnits.tex

```diff
@@ -412,6 +416,81 @@ \subsection{What this document will not do}
 with scaling prefixes (cf \prettyref{sec:scalingprefixes}).
 \end{itemize}
 
+\subsection{VOUnits syntax summary\label{sec:cribsheet}}
+
+This section provides an informal cribsheet for the most important
+rules defined by VOUnits syntax, which is recommended for use
+within the VO.
+More detailed rules and explanations, as well as special cases
+and less obvious consequences,
+are provided in the body of \prettyref{sec:proposal},
+and the syntax is formally defined in \prettyref{appx:vougrammar}.
+\begin{itemize}
+\item Unit strings are printable ASCII characters (\prettyref{sec:encoding})
+\item Unit strings are case-sensitive (\prettyref{sec:encoding})
+\item No whitespace is permitted (\prettyref{appx:vougrammar})
+\item Unit names are, mostly, represented as you would expect.
+      Most unit base symbols required by astronomy data are recognised,
+      including:
+\begin{itemize}
+  \item Basic physical units including
+        \unit{m}, \unit{s}, \unit{g}, \unit{Hz}, \unit{K}
+        (\prettyref{sec:baseUnits})
+  \item Time units including
+        \unit{yr}, \unit{d}, \unit{h}, \unit{min}, \unit{s}
+        ({\em not\/} {\tt day}, {\tt hour}, {\tt sec})
+        (Sects. \ref{sec:astrosymbols}, \ref{sec:baseUnits})
+  \item Angular units including
+        \unit{deg}, \unit{arcmin}, \unit{arcsec}, \unit{mas}
+        ({\em not\/} {\tt degree}, {\tt as};
+        for microarcsecond use \unit{uarcsec} {\em not\/} {\tt uas})
+        (\prettyref{sec:astrosymbols}, \prettyref{tabx:comparUnitAstro})
+  \item Other units useful in astronomy including
+        \unit{Jy}, \unit{pc}, \unit{AU}, \unit{mag},
+        \unit{count}
+        (Sects.\ \ref{sec:astrosymbols}, \ref{sec:other})
+\end{itemize}
+\item Standard SI scaling prefixes
+      (\unit{k}, \unit{M}, \unit{G}, ..., \unit{m}, \unit{u}, \unit{n}, ...)
+      may be used.
+      For $10^{-6}$ the prefix ``\unit{u}''
+      is used instead of the non-ASCII $\mu$
+      (\prettyref{sec:scalingprefixes})
```

Comment:
Although the document does bang on about this in sec:scalingprefixes, it _might_ be worth reminding the reader that these SI prefixes are decimal only, even when referring to bytes.  That might be my hobbyhorse/b̂ete noire, though (a black hobbyhorse?), and I'll go with your judgement about whether it would be a distraction if repeated here.